src/modules/sdk/objections/axios.client.ts

```diff
@@ -54,11 +54,11 @@ export const makeAPICall = async (config: AxiosRequestConfig): Promise<AxiosResp
     logger.error(`ERROR calling API ${config.url} - ${apiErr}`);
     const axiosError = apiErr as AxiosError;
     const { response, message } = axiosError;
-     const thrownError: ApiError = {
-        data: response ? response.data : [],
-        message: message,
-        status: response ? response.status : STATUS_NO_RESPONSE,
-    };
-    throw thrownError
+    const thrownError: ApiError = {
+      data: response ? response.data : [],
+      message: message,
+      status: response ? response.status : STATUS_NO_RESPONSE,
+  };
+  throw thrownError
```

Comment:
The code smell says to immediately throw the expression. Was there a reason it doesn't?

Reply:
I changed it but that just generated a different code smell over you throwing an expression instead of an error so I ended up leaving it

Reply:
Ok, fair enough 👍